### PR TITLE
Hubs and stores

### PR DESCRIPTION
**Pull request by getbez**
here are the model changes and migrations for warehouse and hubs. (remaining work on location and some clean-up for views)

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -9,6 +9,7 @@ capybara-*.html
 /tmp
 /db/*.sqlite3
 /db/*.sqlite3-journal
+/db/schema.rb
```
Comment (asuraphel) [inline .gitignore]: http://stackoverflow.com/questions/6450263/what-is-the-right-approach-to-deal-with-rails-db-schema-rb-file-in-git

**Review by asuraphel — CHANGES_REQUESTED** [review]
Please remove schema.rb from .gitignore

File: .gitignore (hunk 1 above)
Comment (getbez) [reply .gitignore]: good. I did that since it's generated and u guys suggested we ignore it last time. it seems like that's not a good idea after all.

**Comment by getbez** [comment]
have resolved conflicts.

**Pull request merged**